Description: Adds the missing dependency versions no longer inherited from
 the parent pom since the --no-parent flag was used to avoid a dependency
 between librngom-java/libtxw2-java/libcodemodel-java and libjaxb-java
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/jaxb-ri/codemodel/codemodel-annotation-compiler/pom.xml
+++ b/jaxb-ri/codemodel/codemodel-annotation-compiler/pom.xml
@@ -63,6 +63,7 @@
         <dependency>
             <groupId>com.sun.istack</groupId>
             <artifactId>istack-commons-tools</artifactId>
+            <version>debian</version>
             <exclusions>
                 <exclusion>
                     <groupId>org.apache.ant</groupId>
@@ -73,6 +74,7 @@
         <dependency>
             <groupId>org.apache.ant</groupId>
             <artifactId>ant</artifactId>
+            <version>debian</version>
             <scope>provided</scope>
         </dependency>
     </dependencies>
--- a/jaxb-ri/txw/compiler/pom.xml
+++ b/jaxb-ri/txw/compiler/pom.xml
@@ -88,6 +88,7 @@
         <dependency>
             <groupId>org.apache.ant</groupId>
             <artifactId>ant</artifactId>
+            <version>debian</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
--- a/jaxb-ri/external/rngom/pom.xml
+++ b/jaxb-ri/external/rngom/pom.xml
@@ -84,6 +84,7 @@
         <dependency>
             <groupId>relaxngDatatype</groupId>
             <artifactId>relaxngDatatype</artifactId>
+            <version>debian</version>
         </dependency>
         
         <!-- Test -->
